Lab_1 (1).ipynb

```diff
@@ -0,0 +1,948 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 36,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Image(filename='pdf int.JPG') "
```

Comment:
It would probably be good to use latex to display your equations here, Python is capable of it, and it would reduce dependencies on out of code files